From 491096218b0557c7de74f8fb4608c36ea67aa954 Mon Sep 17 00:00:00 2001
From: Robby Cai <robby.cai@nxp.com>
Date: Fri, 14 Oct 2016 14:36:52 +0800
Subject: [PATCH 1210/5242] MLK-13345-1 ARM: dts: csi: correct the base
 address for csi on imx6sll

commit  2cac3237609b65b3fdbe0e56d015172aeede89de from
https://source.codeaurora.org/external/imx/linux-imx.git

correct the base address for imx6sll CSI

Signed-off-by: Robby Cai <robby.cai@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6sll.dtsi |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/imx6sll.dtsi b/arch/arm/boot/dts/imx6sll.dtsi
index f45e705..f1826a9 100644
--- a/arch/arm/boot/dts/imx6sll.dtsi
+++ b/arch/arm/boot/dts/imx6sll.dtsi
@@ -585,7 +585,7 @@
 				reg = <0x020e4000 0x4000>;
 			};
 
-			csi: csi@0x0e8000 {
+			csi: csi@020e8000 {
 				compatible = "fsl,imx6sll-csi", "fsl,imx6s-csi";
 				reg = <0x020e8000 0x4000>;
 				interrupts = <GIC_SPI 7 IRQ_TYPE_LEVEL_HIGH>;
-- 
1.7.9.5

